### PR TITLE
gpsd: update to 3.24

### PR DESCRIPTION
**Pull request by jailbird777**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by Piraty** [comment]
pleae have a branch clean of merge commits, see our docs for help with that

**Comment by jailbird777** [comment]
Sorry about that, accidentally clicked the stupid button in GH. Fixed.

File: srcpkgs/gpsd/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ maintainer="uriahheep <uriahheep@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://gpsd.gitlab.io/gpsd/"
```
Comment (classabbyamp) [inline srcpkgs/gpsd/template:16]:
```suggestion
homepage="https://gpsd.gitlab.io/gpsd/"
changelog="https://gpsd.gitlab.io/gpsd/NEWS"
```

Comment (jailbird777) [reply srcpkgs/gpsd/template:16]: Done!

**Pull request merged**